tutorials/haskell/hspec-golden-test-tutorial/code/package.yaml

```diff
@@ -0,0 +1,54 @@
+name:                hspec-golden-tests
+version:             0.1.0.0
+github:              "githubuser/hspec-golden-tests"
+license:             BSD3
+author:              "Author name here"
+maintainer:          "example@example.com"
+copyright:           "2019 Author name here"
```

Comment:
@Jagl257 Can you update the author name here?

Reply:
Updated Cris